### ISSUE TITLE
Hound's UI has poor accessibility

### ISSUE THREAD

**Issue by tgt** [hound-search/hound#151]
I find the UI of Hound to be extremely difficult to read since its text is very low contrast. I've noted several of the issues I've encountered below. These issues won't just affect the visually impaired, but also those with lower quality screens.

The guidelines laid out in the [Web Content Accessibility Guidelines (WCAG) 2.0](http://www.w3.org/TR/WCAG20/) suggest a contrast ratio of 4.5:1 for AA (3:1 for text of at least 18px) and 7:1 for AAA (4.5:1 for text of at least 18px). I think the default UI should aim for AAA compliance in order to be accessible to as many people as possible.

Examples:
- The text around the search box (e.g. the advanced section, number of results) has a contrast ratio of 2.32:1 (#aaa on #fff), which fails guidelines for text of any size.
- The text of file paths has a contrast ratio of 5.27:1 (#666666 on #f5f5f5), which fails the guideline for text under 18px.
- The text in the Select Repo section of the advanced options fails with a contrast ratio of 5.74:1 (#666 on #fff), which fails the guideline for text under 18px.
- Line numbers also fail with a contrast ratio of 2:1 (#aaa on #eee), which fails guidelines for text of any size.

**Comment by jklein** [comment]
Thanks, this is a good idea. We'd be happy to accept a PR for these changes. Candidly it's not something we are likely to get to in the near term.

**Comment by salemhilal** [comment]
We’ve just merged in a fix for this issue. I’m closing it out, but feel free to re-open it if something’s missing.
